fluxette/js/util.js

```diff
@@ -0,0 +1,21 @@
+// Order-preserving single filter
+export let splicefilter = (iter, pred) => {
```

Comment:
I could use two lines of Array#filter, but that's inefficient, so I wrote a small algorithm instead.